samples/cellular/modem_shell/src/sms/sms_shell.c

```diff
@@ -94,7 +95,9 @@ int sms_shell(const struct shell *shell, size_t argc, char **argv)
 	} else if (!strcmp(argv[1], "recv")) {
 		command = SMS_CMD_RECV;
 	} else {
-		mosh_error("Unsupported command=%s\n", argv[1]);
+		if (strcmp(argv[1], "-h") != 0 && strcmp(argv[1], "--help") != 0) {
+			mosh_error("Unsupported command=%s\n", argv[1]);
```

Comment:
`Unsupported command=%s` is used in a few places, while those where shell subcommands are used print `%s: subcommand not found`. I think it would be nice to align those.

Reply:
I didn't change these because these lines will be removed in an upcoming PR: https://github.com/nrfconnect/sdk-nrf/pull/12848#issuecomment-1780530644